OpenSim/Actuators/VandenBogert2011Muscle.h

```diff
@@ -0,0 +1,568 @@
+                                          double projFiberLength,
+                                          bool normalizedValues) const;
+
+        /**@param fiberVelocity Velocity of the fiber
+           @param fiberLength Fiber Length
+           @param normalizedValues T: The values provided are normalized by the
+           muscle's optimal length, F: Not normalized
+           @returns The projected fiber velocity
+           (will be nomralized if normalizedValue=T)*/
+        double fiberVelocityToProjFibVel (double fiberVelocity,
+                                          double fiberLength,
+                                          bool normalizedValues) const;
+
+        /**@param projFibVel Projected velocity of the fiber
+           @param fiberLength Fiber Length
+           @param projFiberLength projected fiber velocity (in-line with tendon)
+           @param normalizedValues T: The values provided are normalized by the
+           muscle's optimal length, F: Not normalized
+           @returns The fiber velocity
+           (will be nomralized if normalizedValue=T)*/
+        double projFibVelToFiberVelocity(double projFibVel, double fiberLength,
+                                         double projFiberLength,
+                                         bool normalizedValues) const;
+
+        /**@param projFibVel Projected velocity of the fiber
+           @param projFiberLength projected fiber velocity (in-line with tendon)
+           @param normalizedValues T: The values provided are normalized by the
+           muscle's optimal length, F: Not normalized
+           @returns The fiber velocity
+          (will be nomralized if normalizedValue=T)*/
+        double projFibVelToFiberVelocity(double projFibVel,
+                                         double projFiberLength,
+                                         bool normalizedValues) const;
+
+
+        //During tests we want to insure that the anlytical jacobians are
+        // correct.  To do this we can compare them to finite difference
+        //calcualted jacobians.
+
+        /** Calculate Jacobian by finite difference
+              @param y Vector of:
+                        \li 0: projected fiber length
+                        \li 1: muscle activation
+              @param ydot Vector of:
+                        \li 0: projected fiber velocity
+                        \li 1: muscle activation time derivative
+              @param muscleLength The muscle length
+              @param u The muscle control (excitation)
+              @param h The peturbation size of the finite step
+              @returns TBD for now see ImplicitResidual structure*/
+        ImplicitResidual calcJacobianByFiniteDiff(SimTK::Vec2 y,
+                                                  SimTK::Vec2 ydot,
+                                                  double muscleLength,
+                                                  double u, double h ) const;
```

Comment:
I think it would be better to use `excitation` instead of `u` for the variable name, since `u` can be confused with generalized speeds.